v2.0/backup.md

```diff
@@ -149,7 +151,7 @@ AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00';
 
 ~~~ sql
 > BACKUP bank.customers, bank.accounts TO 'azure://acme-co-backup/tables-accounts-customers-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
-AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00');
+AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00';
```

Comment:
Aren't you updating these in another PR? If so, you might want to revert this change so as to not create a conflict for yourself.

Reply:
Hmm..good point. Thanks!